docs/test-cases-and-sections.md

```diff
@@ -90,15 +90,79 @@ This macro maps onto ```TEST_CASE``` and works in the same way, except that the
 * **WHEN(** _something_ **)**
 * **THEN(** _something_ **)**
 
-These macros map onto ```SECTION```s except that the section names are the _something_s prefixed by "given: ", "when: " or "then: " respectively.
+These macros map onto ```SECTION```s except that the section names are the _something_s prefixed by "given: ", "when: " or "then: " respectively. These macros also map onto the AAA or A<sup>3</sup> test pattern (standing either for [Assemble-Activate-Assert](http://wiki.c2.com/?AssembleActivateAssert) or [Arrange-Act-Assert](http://wiki.c2.com/?ArrangeActAssert)), and in this context, the macros provide both code documentation and reporting of these parts of a test case without the need for extra comments or code to do so.
```

Comment:
It has been that in the original already, but the `_something_s` won't render properly.

Also new documentation is written with explicit linebreaks, to avoid situation like this where a single line of diff represents many lines of actual text, which in turn makes reviewing/diffing hard. 😃

Reply:
@horenmar I have attempted to implement your review comments. Let me know what you think.